host-configs/LLNL/quartz-base.cmake

```diff
@@ -33,4 +33,7 @@ set(ENABLE_PETSC OFF CACHE BOOL "Enables PETSc." FORCE)
 # YAPF python formatting
 set(YAPF_EXECUTABLE /usr/gapps/GEOSX/thirdPartyLibs/python/linux-rhel7-x86_64-gcc@8.1.0/python@3.8.5/bin/yapf CACHE PATH "" FORCE)
 
+# Sphinx
+set(SPHINX_EXECUTABLE /usr/gapps/GEOSX/thirdPartyLibs/python/documentation/documentation/bin/sphinx-build CACHE PATH "" FORCE)
```

Comment:
The sphinx executable is really called `sphinx-build` and not `sphinx`?

Reply:
Yeah, I was surprised when I came across this too.  Sphinx builds four executables: sphinx-apidoc, sphinx-autogen, sphinx-build, sphinx-quickstart